lib/ae_mdw/aexn_tokens.ex

```diff
@@ -110,24 +115,33 @@ defmodule AeMdw.AexnTokens do
     end
   end
 
-  defp build_tokens_streamer(%{exact: exact}, state, type, table, cursor) do
+  defp build_tokens_streamer(%{exact: exact}, state, type, order_by, cursor) do
+    sorting_table = Map.fetch!(@sorting_table, order_by)
+
     scope = {
       {type, exact, <<>>},
       {type, exact, Util.max_256bit_bin()}
     }
 
-    do_build_tokens_streamer(state, table, cursor, scope)
+    do_build_tokens_streamer(state, sorting_table, cursor, scope)
   end
 
-  defp build_tokens_streamer(%{prefix: prefix}, state, type, table, cursor) do
+  defp build_tokens_streamer(%{prefix: prefix}, state, type, order_by, cursor) do
+    sorting_table = Map.fetch!(@downcased_sorting_table, order_by)
     prefix = String.slice(prefix, 0, @max_sort_field_length)
 
+    cursor =
+      if not is_nil(cursor) do
+        {type, symbol_name, pubkey} = cursor
+        {type, String.downcase(symbol_name), pubkey}
+      end
```

Comment:
This block isn't needed since these values are the values obtained from the downcased table keys

Reply:
The cursor is formed by using the keys from the table which are always already downcased

Reply:
Doesn't the cursor come from the serialization of `do_build_tokens_streamer(state, table, cursor, scope)` where the `table` is one of these downcased tables?

Reply:
At the end of that function the results are from the AexnContract table, so after it when the stream is being paginated it returns the cursor based on that table

Reply:
I see that now! You're right. We should change it, though, since we don't need to `fetch!` records to form the pagination next/prev urls, that's not necessary. I'll update it later. Thanks for clarification 🙏